01-node-tutorial/answers/content/result-mind-grenade.txt

```diff
@@ -0,0 +1 @@
+THIS IS AWESOME : Hello this is first text file Hello this is second text fileTHIS IS AWESOME : Hello this is first text file Hello this is second text fileTHIS IS AWESOME : Hello this is first text file Hello this is second text file
```

Comment:
Was this file created by one of the scripts numbered 01-12 in the folder `01-node-tutorial/answers` ? I don't see the code that generates or uses this file. I think it may be for a later lesson but just wanted to double check if you added it here intentionally

Reply:
I guess this code is generared later, in promises-async-syntax.js (folder 01-node-tutorial/answers). 
I 'm not sure why it appeared in this commit ..
 should i merge now this branch with the main?